documentation/modules/creating-vddk-image.adoc

```diff
@@ -10,6 +10,8 @@
 
 You must download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry. You need the VDDK init image path in order to add a VMware source provider.
```

Comment:
```suggestion
You need to download the VMware Virtual Disk Development Kit (VDDK), build a VDDK image, and push the VDDK image to your image registry.
```
unrelated to the original intent of this PR: we made VDDK optional in 2.5 but forgot to update this section. also in line 9 above we should state that we support and recommend using VDDK to accelerate the migration but don't require it